exporter/exporterhelper/metrics.go

```diff
@@ -66,11 +67,18 @@ func (req *metricsRequest) Export(ctx context.Context) error {
 }
 
 func (req *metricsRequest) ItemsCount() int {
-	return req.cachedItemsCount
+	return req.md.DataPointCount()
 }
 
-func (req *metricsRequest) setCachedItemsCount(count int) {
-	req.cachedItemsCount = count
+func (req *metricsRequest) Size(sizer sizer.MetricsSizer) int {
```

Comment:
No need to be public, rename to `size`

Reply:
It needs to be public because it used in `metrics_batch.go` and `metrics_batch_tests.go`

Reply:
And?

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/12595

Reply:
Oh, ok sorry. I will change it.